client-dotnet/src/Controller.Api/dto/ActionDto.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace Controller.Api {
+    public class ActionDto {
+
+        /**
+         * The index of this action in the test.
+         * Eg, in a test with 10 indices, the index would be
+         * between 0 and 9
+         */
+        public int? Index { get; set; }
```

Comment:
regarding all DTOs, not just this one.
it "might" be that this not going to work, as JSON is case-sensitive, whereas here all fields are declared in capital-case.
Ie, it might happen that this will fail when using EM, as EM expect all these fields not in capital-case.
However, before refactoring, should check if indeed it is the case

Reply:
Property names should be in PascalCase based on capitalization conventions in C#. However, camelCase is supported for json. DTOs are returned in camelCase by default (configurable) and also you can send a DTO in the body of a POST using either camelCase or PascalCase.